### ISSUE TITLE
How to use @swc/register with some typescript that imports non-TS code?

### ISSUE THREAD

**Issue by zxti** [swc-project/swc#460]
How do I use swc (and specifically swc/register) to just run some typescript code in node?

For instance, here is a .swcrc I've been trying to get to work:

```
{
  "jsc": {
    "target": "es2015",
    "parser": {
      "syntax": "typescript",
      "tsx": true,
      "decorators": true
    }
  },
  "module": {
    "type": "commonjs"
  }
}
```

If the typescript code depends on any libraries, I immediately run into failures.  Simply running this does not work:

```
node -r @swc/register/lib/index.js -r lodash
```

You can replace lodash with typescript, webpack, request, or various other libraries.

If you remove the .swcrc, then other libraries can be imported, but not lodash (still fails with `TypeError: memoize.Cache is not a constructor`).  But without the .swcrc I cannot run any typescript.

**Comment by kdy1** [comment]
I'm investigating on this, and seems like

**Comment by kdy1** [comment]
I fixed the issue in @swc/register v0.1.3. Closing.
